packages/fxa-auth-server/lib/routes/subscriptions/paypal.ts

```diff
@@ -82,5 +162,30 @@ export const paypalRoutes = (
       handler: (request: AuthRequest) =>
         paypalHandler.getCheckoutToken(request),
     },
+    {
+      method: 'POST',
+      path: '/oauth/subscriptions/active/new-paypal',
+      options: {
+        auth: {
+          payload: false,
+          strategy: 'oauthToken',
+        },
+        response: {
+          schema: isA.object().keys({
+            subscription: validators.subscriptionsStripeSubscriptionValidator,
+            sourceCountry: validators.subscriptionPaymentCountryCode.required(),
+          }),
+        },
```

Comment:
The validation here still throws an error and returns a 500 response for me on the client. I only get a 200 when I add `failAction: 'log',` as we had to do for the demo. Can we fix this before merging else file a follow-up to fix it next?